tests/unit/test_hnsw_tiered.cpp

```diff
@@ -2889,7 +2889,7 @@ TYPED_TEST(HNSWTieredIndexTest, switchWriteModes) {
                          .metric = VecSimMetric_L2,
                          .multi = TypeParam::isMulti(),
                          .M = 32,
-                         .efRuntime = 2 * n};
+                         .efRuntime = 3 * n};
```

Comment:
why the change?

Reply:
see https://redis.slack.com/archives/C03QDNA8K0B/p1690090714770389